index.html

```diff
@@ -228,5 +228,67 @@ <h2>Skills</h2>
         </li>
       </ul>
     </section>
+
+    <section class="get-in-contact">
+      <p class="primary-text">
+        I'm always interested in hearing about new projects, so if you'd like 
+        to chat please get in touch.
+      </p>
+
+      <form action="https://formspree.io/f/xvolaban" method="post">
+        <label class="input">
+          <input type="text" name="user_name" maxlength="30" placeholder="Full name" required>
+        </label>
+
+        <label class="input">
+          <input type="email" name="email-address" placeholder="Email address" required>
+        </label>
+
+        <label class="textarea">
+          <textarea name="message" maxlength="500" placeholder="Enter text here" required></textarea>
+        </label>
+
+        <button class="button" type="submit">Get in touch</button>  
+      </form>
+
+      <nav class="socials">
+        <ul>
+          <li class="social">
+            <a href="#"
+              ><img src="images/icons/twitter-icon.svg" alt="Twitter Profile"
+            /></a>
+          </li>
+
+          <li class="social">
+            <a href="#"
+              ><img src="images/icons/linkedin-icon.svg" alt="LinkedIn Profile"
+            /></a>
+          </li>
+
+          <li class="social">
+            <a href="#"
+              ><img src="images/icons/medium-icon.svg" alt="Medium Profile"
+            /></a>
+          </li>
+
+          <li class="social">
+            <a href="#"
+              ><img src="images/icons/github-icon.svg" alt="Github Profile"
+            /></a>
+          </li>
+
+          <li class="social">
+            <a href="#">
+              <img
+                src="images/icons/angellist-icon.svg"
+                alt="Angellist Profile"
+              />
+            </a>
+          </li>
+        </ul>
+      </nav>
+
+      <div class="my-email">ghooryahya@gmail.com</div>
```

Comment:
i don't think you are supposed to put text directly in a div element. it should be a p